pandas/core/indexes/range.py

```diff
@@ -164,6 +168,8 @@ def _simple_new(cls, start, stop=None, step=None, name=None,
         for k, v in kwargs.items():
             setattr(result, k, v)
 
+        result._range = range(result._start, result._stop, result._step)
```

Comment:
we could actually remove the _start, _stop, _step properties as well?

Reply:
Yes, I'm planning to do that in an upcoming PR.

Python3's ``range`` accept slicing ranges, which Python2's ``xrange`` didn't, so this refactoring will also allow dropping doing custom slicing operations in RangeIndex.